semantic-release/action.yaml

```diff
@@ -28,6 +28,18 @@ runs:
       with:
         node-version: "14"
 
+    - name: Build the Code
+      run: |
+        if exit | yarn run 2>&1 | grep "\- build"
+        then
+          yarn
+          yarn build
+        else
+          echo "No build script defined - ignoring"
+        fi
+      working-directory: ./
+      shell: bash
+
```

Comment:
Can we re-use the already built artifact from the other composite action? This way we could avoid building the source twice.

Reply:
I don't think, that this can be accomplished that easily since these two actions are not related

Reply:
What about adding an extra input flag so that the code is only build or certain projects.